frontend/src/views/Simulator/SettingsView.vue

```diff
@@ -422,5 +468,23 @@ const handleCreateNewValidator = async () => {
         </button>
       </div>
     </Modal>
+    <Modal :open="resetStorageModalOpen" @close="closeResetStorageModal">
+      <div class="flex flex-col">
+        <div class="flex justify-between">
+          <div class="text-xl">Reset Simulator Storage</div>
+        </div>
+        <div class="flex justify-between font-bold bg-slate-100 p-2 mt-4">
+          Are you sure you want to reset the simulator storage?
```

Comment:
@bradleySuira please add more information about what this will do to all your contracts and stuff.
As far as I understand, it will remove all the users contracts as well right? Is there a way to avoid it? by setting some parameter in the database, for example?

Reply:
fixed as requested